src/sensor/pingsensor.h

```diff
@@ -198,18 +198,18 @@ class PingSensor : public Sensor
     void writeMessage(const ping_message& msg) const;
 
     QString _ascii_text;
-    uint8_t _device_revision;
-    uint8_t _device_type;
-    uint8_t _dstId;
-    uint8_t _firmware_version_major;
-    uint8_t _firmware_version_minor;
-    uint8_t _firmware_version_patch;
-    int _lostMessages;
+    uint8_t _device_revision {0};
```

Comment:
Usually the uniform initialization is done with the braces next to the variable and not with a space between it, thing it like a constructor call.  
https://en.cppreference.com/w/cpp/language/list_initialization

Reply:
aha